test/e2e/node_exporter_test.go

```diff
@@ -83,3 +83,54 @@ nodeExporter:
 	}
 
 }
+
+func TestNodeExporterCollectorDisablement(t *testing.T) {
```

Comment:
can you add a check that the netdev collector's metrics are present by default?

Reply:
of course :) added a check for multiple metrics of `netdev` collector.